Dockerfile

```diff
@@ -1,25 +1,19 @@
-FROM node:16.19.0-alpine
+FROM amazon/aws-lambda-nodejs:20
```

Comment:
This is the docker image used in the Lambda node runtime. The reason we're switching to this (you'll see the same change in the `async_operations` Dockerfile) is that it provides the node version we're on and other Lambda runtime things like the aws-sdk. The aws-sdk v3 is like 450ish packages - instead of trying to maintain that full list on our own we can just use what AWS already packages for us.

Reply:
Awesome. Thanks for the context!